# How many faults the watchdog has picked up
uint8 faults
